concourse/scripts/pipecleaner.py

```diff
@@ -40,6 +44,27 @@ def load_pipeline(self, filename):
         raw = re.sub('\{\{.*?\}\}', 'DUMMY', raw)
         return yaml.load(raw)
 
+    def shellcheck(self, shell, job, task, args):
+        if not self.shellcheck_enabled:
+            return
+
+        if args[-2] != '-c':
+            print "Odd args " + job + "." + task
+            print args
+            return
+
+        script = tempfile.NamedTemporaryFile(
```

Comment:
That seems sensible - I wouldn't worry too much about parsing it for now, simply dumping stdout from shellcheck if it exits nonzero is reasonable.